src/tools/interfaces.md

```diff
@@ -16,8 +16,6 @@ The [Mozilla Growth & Usage Dashboard](https://gud.telemetry.mozilla.org/) (GUD)
 
 The [Glean Aggregated Metrics Dashboard](https://glam.telemetry.mozilla.org/) (GLAM) is an interactive dashboard that is Mozilla’s primary self-service tool for examining the distributions of values of specific individual telemetry metrics, over time and across different user populations. It is similar to GUD in that it is meant to be usable by everyone; no specific data analysis or coding skills are needed. But while GUD is focused on a relatively small number of high level, derived product metrics about user engagement (e.g. MAU, DAU, retention, etc) GLAM is focused on a diverse and plentiful set of probes and data points that engineers capture in code and transmit back from Firefox and other Mozilla products.
 
-As of this writing, GLAM is in active development but is already usable.
-
 ### Telemetry Measurement Dashboard
 
 The [Telemetry Measurement Dashboard](https://telemetry.mozilla.org/new-pipeline/dist.html) (TMO) site is the
```

Comment:
Can you update the description here with the adjustments you made in the GLAM article (e.g. removing the part about GLAM not being production ready)?